### ISSUE TITLE
Building the app_run CI container image fails

### ISSUE THREAD

**Issue by jasonf-trustgrid** [microsoft/CCF#4187]
**Describe the bug**
Building the app_run CI container image fails because the [latest release feed](https://api.github.com/repos/microsoft/ccf/releases/latest) has multiple .deb assets.

> #13 110.1 fatal: [localhost]: FAILED! => {"changed": false, "msg": "Failure downloading https://github.com/microsoft/CCF/releases/download/ccf-2.0.7/ccf_2.0.7_amd64.deb\nhttps://github.com/microsoft/CCF/releases/download/ccf-2.0.7/ccf_unsafe_2.0.7_amd64.deb, An unknown error occurred: URL can't contain control characters. '/microsoft/CCF/releases/download/ccf-2.0.7/ccf_2.0.7_amd64.deb\\nhttps://github.com/microsoft/CCF/releases/download/ccf-2.0.7/ccf_unsafe_2.0.7_amd64.deb' (found at least '\\n')"}

**To Reproduce**
```sh
git clone git@github.com:microsoft/CCF.git
docker build -t localtesting/ccf-app-run -f docker/app-run .
```

**Expected behavior**
The image should build correctly.

**Environment information**
N/A

**Additional context**
The code/script causing the issue is currently located at https://github.com/microsoft/CCF/blob/release/2.x/getting_started/setup_vm/roles/ccf_install/tasks/deb_install.yml#L8

**Comment by jasonf-trustgrid** [comment]
I have been able to work around the issue by adding ` | grep -v unsafe` to the end of [the line](https://github.com/microsoft/CCF/blob/release/2.x/getting_started/setup_vm/roles/ccf_install/tasks/deb_install.yml#L8)

**Comment by jumaffre** [comment]
@jasonf-trustgrid: Thank you for noticing and raising this. This is something we've recently fixed, but only on the `main` branch (see https://github.com/microsoft/CCF/pull/4017). As you're looking at a `2.x` release (specifically `2.0.7`), which is cut from the `release/2.x` branch, the fix isn't present there. 

I've now backported the fix to the `release/2.x` branch (https://github.com/microsoft/CCF/pull/4189) and this will be available from the next release (`2.0.8`). In the meantime, the workaround you've identified should work fine.
